switchmap/poller/snmp/async_snmp_info.py

```
@@ -0,0 +1,204 @@
+        """Add data from source to target dict. Both dicts must have two keys.
+
+        Args:
+           source: Source dict
+        target: Target dict
+
+        Returns:
+            target: Aggregated data
+
+        """
+        # Process data
+        for primary in source.keys():
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)